in-development/CMR-STAC-Search.qmd

```diff
@@ -0,0 +1,197 @@
+---
+title: "Getting Started with NASA's CMR-STAC API in R"
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+knitr::opts_chunk$set(comment = NA)
+```
+
+**This tutorial demonstrates how to interact with CMR-STAC in R.**
+
+This tutorial will teach you how to navigate and explore NASA's Common Metadata Repository (CMR) SpatioTemporal Asset Catalog (STAC) to learn about the datasets available through LP DAAC Cumulus cloud archive.
+
+------------------------------------------------------------------------
+
+### Topics Covered in this Tutorial
+
+1.  **Introduction to STAC and the CMR-STAC API**\
+    1a. What is STAC?\
+    1b. What is the CMR-STAC API?\
+2.  **Get started & searching with CMR-STAC**\
+    2a. CMR-STAC API\
+    2b. STAC Collection\
+    2c. STAC Item\
+    2d. Assets
+
+------------------------------------------------------------------------
+
+### Required packages
+
+-   **Required packages:**
+
+    -   `jsonlite`: change the format of returned content
+    -   `DT`: make returned information more readable
+    -   `rstac`: interact with STAC catalogs
+    -   `devtools`: install development version of packages
+
+Run the cell below to identify any missing packages to install, and then load all of the required packages.
+
+```{r}
+packages <- c('jsonlite','DT', 'rstac', 'devtools')
+new.packages <- packages[!(packages %in% installed.packages()[,"Package"])]
+if(length(new.packages)) install.packages(new.packages, repos='http://cran.rstudio.com/') else print('All required packages are installed.')
+invisible(lapply(packages, library, character.only = TRUE))
+```
+
+For listing the various STAC catalogs associated with NASA CMR, we'll install the development version of the `earthdatalogin` package in the following cell. The development version contains functions that will be utilized in section 2a.
+
+```{r}
+devtools::install_github("boettiger-lab/earthdatalogin")
+library("earthdatalogin")
```

Comment:
I'm not sure if it will matter but I got a warning 
```



Attaching package: 'earthdatalogin'

The following object is masked from 'package:rstac':

    collections_fetch

```

Will probably be ok, but particularly if you do `earthdatalogin::` syntax